content/docs/2.0/operate.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Operate"
++++
+
+## Prometheus Exporter Metrics
+
+The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`.  The metrics collected in the Metrics Adapter are only active when the HPA is active (> 0 replicas).
+
+The following metrics are being gathered:
+
+- `keda_metrics_adapter_scaler_error_totals` - The total number of errors encountered for all scalers
+- `keda_metrics_adapter_scaled_object_error_totals`- The number of errors that have occurred for each scaled object
+- `keda_metrics_adapter_scaler_errors` - The number of erros that have occurred for each scaler
+- `keda_metrics_adapter_scaler_metrics_value`- The current value for each scaler's metric that would be used by the HPA in computing the target average.
```

Comment:
This is more the metric for a given scaled object then, no?

Reply:
If a scaled object that multiple scalers there would be separate metrics by scaler so I thought calling it scaler would be more appropriate. I can rename it. If so, it would also make sense to rename `keda_metrics_adapter_scaler_errors` to `keda_metrics_adapter_scaled_object_metric_errors` or something

Reply:
My remark was actually more for `keda_metrics_adapter_scaled_object_error_totals` or is that what you are talking about?